.github/workflows/nodejs.condo.ci.yml

```diff
@@ -6,7 +6,8 @@ on:
       - main
 
   # NOTE: change pull_request_target to pull_request if you want to change the CI in your branch
-  pull_request_target:
+#  pull_request_target:
```

Comment:
is it just so that tests are collected?

Reply:
To test collection logic, since dispatching worklflows are also inherit target behaviour...And I need to change it

I'll rollback this as soon as I test it